shell.nix

```diff
@@ -2,16 +2,19 @@
 , defaultLv2Plugins ? false
 , lv2Plugins ? []
 , releaseMode ? false
+, enableKeyfinder ? true
+, buildType ? "auto"
 }:
 let inherit (nixroot) stdenv pkgs lib
     chromaprint fftw flac libid3tag libmad libopus libshout libsndfile lilv
     libusb1 libvorbis libebur128 pkgconfig portaudio portmidi protobuf qt5 glib
     rubberband sqlite taglib soundtouch vamp opusfile hidapi upower ccache git
-    libGLU x11 lame lv2 makeWrapper pcre utillinux libselinux libsepol
+    libGLU x11 lame lv2 makeWrapper pcre utillinux libselinux libsepol serd
     libsForQt5
     clang-tools
     cmake
     fetchurl
+    fetchgit
```

Comment:
Please sort these alphabetically. Also add a new line for `serd` and sort it appropriately.

I started this "one line per package" convention in order to minimize git diffs and reduce the probability of merge conflicts.

Reply:
I'm not so sure if 50 lines there and 40 lines in the build input it nice style for that. I tried to order it thematically

Reply:
I don't know, why we `inherit` them, anyway.

Reply:
i cleaned up that mess